	<section class="Engineering">
		<div class="nav-items-engg">
			<ul>
				<a href="#Introduction"><li>Introduction</li></a>
				<a href="#Cloning"><li>Plasmid Design and Cloning</li></a>
				<a href="#Proteins"><li>Protein Expression & Purification</li></a>
        		<a href="#Assay"><li>Characterisation and Assay</li></a>
				<a href="#Conclusions"><li>Conclusion/Future Directions</li></a>
				<a href="#References"><li>References</li></a>
			</ul>
		</div>				

		<div class="content-box">
			<div class="content-info">
				<h1 id="Introduction">Introduction</h1>
					<p>Small heat shock proteins (sHSP) are ATP-independent molecular chaperones which associate with misfolded proteins to prevent further aggregation when the cell is under thermal stress. The functional activity of sHSP is therefore classified as a “holdase” chaperone. (1) Proteins are functional when they are soluble in their environments. However, thermal stress can cause a protein to unfold and expose its hydrophobic core to the solvent surface. Subsequent interactions between surface-exposed hydrophobic cores and proteins can result in their aggregation to form large non-functional complexes. sHSPs prevent this aggregation process by binding to the exposed hydrophobic residues, and thereby prevent the proteins from becoming insoluble.(2, 3) In addition, the sHSP can facilitate the transfer of the denatured substrate to other chaperones, such as chaperonins, for ATP-dependent protein refolding. (4) sHSPs are ubiquitous across organisms as these chaperones have large substrate binding capacities, making them a suitable candidate in reversing heat shock. In addition, sHSP genes can be upregulated 1000 fold upon exposure to cellular stress, consequently increasing the activity of ATP-dependent chaperones by 80%. (5)</p> 

					<p>It was hypothesised that transforming HSP22E and 22F from Chlamydomonas reinhardtii (C. reinhardtii) into Symbiodinium would increase the thermal tolerance of the latter microorganism. C. reinhardtii is a green algae which is stable at 42oC, (5) well above the bleaching temperature threshold of Great Barrier Reef. (6,7) The HSP22E and 22F target the chloroplast of C. reinhardtii and protect the photosystems from reactive oxygen species produced by thermal shock. (5) We hypothesize that the expression of HSP22E and 22F in Symbiodinium sp. will improve the organisms thermotolerance and reduce its expulsion from coral tissue - therefore reducing the rate of coral bleaching. </p>

					<p>Due to limited laboratory access resulting from COVID-19 restrictions, lab work focused on characterising the HSP 22E and 22F chaperones through recombinant expression in a standard laboratory E. coli chassis. Plasmid assembly and cloning into competent strains enabled recombinant protein expression and purification of HSP22E and 22F. Functional assays demonstrated the ability of these chaperones to successfully reduce protein aggregation in thermal stress conditions. </p>

					<img src="/assets/images/Engineering/workflow.png">
					<p><b>Experimental Workflow</b></p>

					<p>The design, test, build paradigm was utilised at multiple levels during this process. Following project design, experimental goals to characterise HSP22E and HSP22F were defined. This informed the plasmid design and subsequent ‘build’ phases in the form of DNA cloning and purification followed by the ‘test’ phases with characterisation experiments. This paradigm was also applied in order to adapt protocols to lab challenges and optimise conditions for purifications and assays. The conclusions drawn from lab work this year will go on to inform future work in Phase II of the project in 2021.</p>

		
				<h1 id="Cloning">Plasmid Design and Cloning</h1>
				<h2>Plasmid Design with HSP22E and HSP22F Inserts</h2>
				<p>DNA sequences for our genes of interest (HSP22E and HSP22F) were obtained from Genbank. Gene constructs for each were designed with the following features:</p> 
					<ol>
						<li>Forward and reverse overhangs were added onto both 5’ and 3’ ends of the gene sequences to enable Gibson assembly. The overhangs were complementary to the pET-19b plasmid backbone.</li>
						<ul>
							<li>Fwd: 5’ CGGCTGCTAACAAAGCCCGA 3’</li>
							<li>Rev: 5’ CTTTAAGAAGGAGATATACC 3’</li>
						</ul>
						<li>6x His-tag and GSG linker - The His-tag consisted of six histidines, which enable protein purification using an immobilised metal affinity chromatography column. The GSG linker allowed for protein folding without interference by the 6xHis-tag.</li>
						<ul>
						<li>5’ GGCTCCGGCGGACATCATCATCATCACCATTAA 3’</li>
						</ul>
					</ol>

				<p>As the HSP22 genes were obtained from eukaryotic C. reinhardtii, gene constructs were codon optimised for E. coli using the IDT Codon Optimisation Tool. DNA g-blocks were synthesised by Integrated DNA Technologies (IDT).</p>

				<p>Constructs were designed to be inserted into the pET-19b plasmid backbone (figure 2), a standard E. coli protein expression vector. The plasmid possesses an ampicillin resistance gene to allow for selection of successfully transformed colonies. It also utilises the T7 expression system under the control of a modified Lac operon. The LacI gene produces a lac repressor, which binds to the lac operator. This has been placed downstream of the T7 promoter. Only when IPTG is added is the lac repression relieved, allowing for the T7 polymerase to bind to the T7 promoter and begin transcription of the inserted gene. (8) The DE3 strains have been modified with a copy of the phage T7 RNA polymerase gene and thus were chosen as our experimental chassis. This expression system was chosen as it is inducible and the T7 mechanism ensures almost no leakage.</p>

				<div class="figures">
					<img src="/assets/images/Engineering/HSP22E-in-pET-19b-sequence.png">
					<p><b>Figure 2:</b> Diagram of pET-10b plasmid with the insert HSP22E. A similar plasmid was designed for the HSP22F insert. Ampicillin resistance gene (AmpR) and Lac repressor (LacI) can be seen alongside their respective promoters. Gene was inserted with designed overhangs in between T7 promoter and terminator. Image produced on Benchling. </p>
				</div>

				<h2>Plasmid Assembly and Transformation</h2>
				<p>The two gene constructs were inserted into linearised pET-19b plasmid backbone using Gibson assembly to form two different plasmid products. Plasmids were transformed into chemically competent E. coli BL-21 using heat shock. The cells were plated onto agar plates containing ampicillin, alongside two negative controls, and incubated overnight. HSP22E transformation plates were found to have 7 viable colonies, while HSP22F transformation plates were found to have 10 viable colonies. 5 viable colonies from each were selected. Patch plates and glycerol stocks were prepared to preserve the transformed cells. </p>

				<p>A colony PCR was performed to identify colonies containing the assembled plasmid with an insert. The PCR products were visualised via DNA gel electrophoresis (figure 3) to identify which colonies contained products of specific sizes that correlated with the insert. Successful Gibson assembly and transformation was indicated in colony 5 (C5) for HSP22E and colonies 2-5 (C2-5) for HSP22F, seen in the solid boxed at around 850 bp. Colonies 1 for both HSP22E and HSP22F showed slightly higher bands, seen in dashed boxes at around 1000 bp. These were also investigated in further steps.</p>

				<div class="figures">
					<img src="/assets/images/Engineering/PCR gel.png">
					<p><b>Figure 3:</b> DNA gel electrophoresis showing expected PCR products with HSP22E and HSP22F inserts amplified from designed plasmid construct. Solid boxes indicate bands at the expected 850 bp size, while dashed boxes indicate slightly larger 1000 bp inserts, upon comparison to the 1 kb+ DNA ladder. </p>
				</div>

				<h2>Plasmid Purification and Sequencing</h2>
				<p>To verify that desired gene constructs were successfully transformed, colonies were chosen for sequencing. Colonies 1 and 5 (C1, C5) were selected for the HSP22E assembly whilst colonies 1, 2 and 4 (C1, C2, C4) were chosen for the HSP22F assembly. These colonies were grown up in larger cultures overnight. Plasmid DNAs were extracted and purified using the Qiagen QiaSpin Miniprep Kit. The concentration and purity of each DNA sample was measured on a ThermoFisher NanoDrop Spectrophotometer. Results showed in Table 1 below indicated DNA of adequate quality and little contamination as inferred from the A260/A280 ratio over 1.8 and A260/A230 ratio over 2.</p>

				<p><b>Table 1:</b> DNA concentration of plasmid purification as obtained by NanoDrop spectrophotometry. </p>
				    <table id="protocols">
				      <thead>
				        <tr>
				          <td><b>Sample</b></td>
				          <td><b>Concentration (ng/μL)</b></td>
				          <td><b>A260/A280</b></td>
				          <td><b>A260/A230</b></td>
				        </tr>
				      </thead>

				      <tr>
				        <td>HSP22F C1</td>
				        <td>39</td>
				        <td>2.01</td>
				        <td>1.85</td>
				      </tr>

				      <tr>
				      	<td>HSP22F C2</td>
				        <td>40.5</td>
				        <td>1.96</td>
				        <td>2.11</td>
				      </tr>

				      <tr>
				      	<td>HSP22F C4</td>
				        <td>35.4</td>
				        <td>1.9</td>
				        <td>1.84</td>
				      </tr>

				      <tr>
				        <td>HSP22E C1</td>
				        <td>37.1</td>
				        <td>1.84</td>
				        <td>1.84</td>
				      </tr>

				      <tr>
				        <td>HSP22E C5</td>
				        <td>42.7</td>
				        <td>1.96</td>
				        <td>1.94</td>
				      </tr>
				    </table>

				<p>Sanger sequencing was performed at the Ramaciotti Centre for Genomics (UNSW, Sydney), with both forward and reverse sequencing reactions submitted. Sequencing results analysis using the alignment tool on Benchling indicated high sequencing homology for all colonies. Sequences were identical to the original gene construct for HSP22E C5 and HSP22F C2 and C4. HSP22E C1 and HSP22F C1 had a high level of mismatched bases compared to the original sequence and were omitted from future steps.</p> 

				<h2>Discussion</h2>
				<p>The cloning process was successful and recombinant E. coli containing our designed plasmids were obtained. This was verified using PCR visualisation, NanoDrop spectrophotometer and Sanger sequencing. A number of controls were used throughout this to ensure that only successful transformants were identified for future steps. When transforming the cells, two negative controls were also conducted. In these, water or plasmid backbone only were added to competent cells instead of the Gibson reaction. The water only negative control plate indicated zero colonies, confirming that colonies on other plates weren’t due to contamination. Plasmid backbone only control plates grew six colonies, indicating that there were background circularised plasmid present. This is likely the explanation for the multiple bands at unexpected sizes seen in the DNA gel electrophoresis (C2-4 for HSP22E).</p> 

				<p>Additionally, when sequencing to confirm identity of the gene inserts, both a forward sequencing and reverse sequencing reaction was submitted for each colony. This ensured that sequencing errors would not affect results. This proved helpful when sequencing results for HSP22F Colony 2 (reverse primer) indicated the addition of a guanine base. However, this was not present in the forward sequencing reaction, and further inspection of the sequencing chromatogram confirmed that it was only a base calling error.</p>

		        <h1 id="Proteins">EXPRESSION AND PURIFICATION</h1>
		        <h2>Expression of HSP22E and HSP22F</h2>
				<p>Overnight cultures for HSP22E C5, HSP22F C2 and HSP22F C4 were made. Samples were inoculated into fresh Luria broth and grown until  OD600 reached 0.6. The culture was divided into 2 smaller cultures before IPTG was added to induce expression of the HSP22 proteins. These E. coli cultures were grown overnight at 20oC before cell harvesting via centrifugation and freezing for future work.</p>

				<p>A protein expression check was performed using the BugBuster cell lysis agent to separate soluble and insoluble protein fractions. SDS-PAGE was used to visualise the protein bands and compare non-induced samples from induced samples, as shown in Figure 4. Lack of visible bands for insoluble fractions indicated that most protein was soluble. Expression seemed to be successful for all cultures as a new, comparatively darker band can be seen in the soluble after inductions fractions at around 37 kDa. Though this is higher than our expected band at 22 kDa, previous literature affirmed that recombinant sHSPs often run at higher bands. (9, 10)</p>

				<div class="figures">
					<img src="/assets/images/Engineering/4. expression gels.png">
					<p><b>Figure 4:</b> SDS PAGE showing results from Bugbuster protein expression check. A) Includes HSP22E C5 batch 1 and 2 along with HSP22F batch 1. B) Includes HSP22F batch 2 and HSP22F batch 1 and 2. Pink boxes indicate darker bands in post-induction soluble samples at the expected 37 kDa size of HSP22E and F. Samples were visualised in comparison to the Thermo Scientific Spectra Multicolor High Range Protein Ladder.</p>
				</div>

				<h2>Protein Purification of HSP22E and HSP22F</h2>
				<p>To verify expression, cell cultures were then lysed through sonication and the His-tagged HSP22E and HSP22F were purified using HisPur Ni-NTA resin (Thermofisher) in a gravity column. The purification was repeated three times (Table 2), each with protocol optimisations to improve purity of the fractions obtained.</p>


					<p><b>Table 2.</b> Purification methods of HSP22E and HSP22F via HisPur Ni-NTA resin. All three attempts with various binding and elution buffer concentrations and purification methods.</p>
				    <table id="protocols">
				      <thead>
				        <tr>
				          <td><b></b></td>
				          <td>Binding Buffer</td>
				          <td>Elution Buffer</td>
				          <td>Purification method</td>
				        </tr>
				      </thead>

				      <tr>
				        <td>Purification #1</td>
				        <td>10 mM imidazole</td>
				        <td>100 mM - 500 mM imidazole gradients</td>
				        <td>Entirely in gravity column</td>
				      </tr>

				      <tr>
				      	<td>Purification #2</td>
				        <td>10 mM imidazole</td>
				        <td>50 mM - 150 mM imidazole gradients</td>
				        <td>Binding steps with centrifugation Elution steps in gravity column</td>
				      </tr>

				      <tr>
				      	<td>Purification #3</td>
				        <td>20 mM imidazole</td>
				        <td>50 mM - 500 mM imidazole gradients</td>
				        <td>Binding steps with centrifugation</td>
				      </tr>

				      <tr>
				        <td></td>
				        <td></td>
				        <td></td>
				        <td>Elution steps in gravity column</td>
				      </tr>
				    </table>

				<div class="figures">
					<img src="/assets/images/Engineering/5. purification gels.png">
					<p><b>Figure 5:</b> SDS PAGE showing results from protein purification for HSP22E and HSP22F. Samples were visualised against Thermo Scientific Spectra Multicolor High Range Protein Ladder, seen in lane 1. Labelling for lanes 6 through to 15 indicate the gradient concentrations of imidazole within the elution buffer. The boxes indicate the fractions pooled together to obtain a larger sample of protein. The lower box indicates the bands with expected size at around 37 kDa, whilst the higher boxes indicate the larger 70 kDa size bands. </p>
				</div>

				<p>Purification fractions were visualised using SDS-PAGE. In the third attempt, HSP22E purification produced fractions with high concentration and greater purity in lanes 11, 12, 13 and 14. This can be seen in Figure 5, with a band at around 37kDa and 70kDa, indicating homodimerisation. The HSP22F purification similarly showed a high concentration of protein, but with greater background noise. The column used for this purification was the same as that of the highly contaminated HSP22E purification in attempt #2, indicating that it was a problem within the column filter causing contamination with other unwanted proteins. Regardless, both purifications were successful, with sufficient pure protein for experimental assays. The fractions produced from 250 mM to 400 mM imidazole in the elution buffer (lanes 11-14) were pooled to obtain a larger sample.</p>

				<h2>Discussion</h2>
				<p>During protein expression, the initial large 500 mL culture was grown at 37oC on a large shaking incubator. However, following induction with IPTG, there were no large shaking incubators available to be set at 20oC, which is required for soluble sHSP expression overnight. Instead the 500 mL culture was split into 2, 250 mL cultures which had their protein expression checked separately using BugBuster to ensure both cultures showed good expression.</p>

				<p>Purification protocols were optimised through multiple iterations. The first purification indicated high levels of contamination at most elution fractions. Given this, in the second purification, the lysate incubation steps and wash steps were conducted using centrifugation of the affinity resin rather than gravity columns. This provided a greater surface area and incubation time for HSP22 proteins to bind to the resin. The elution steps were conducted in the gravity column as normal. Additionally, elution buffer gradients were taken at a lower range from 50 mM - 150 mM to knock off contaminating proteins that bind to the column at a lower affinity. The final elution fraction was 500 mM to elute the tightly binding HSP22. While this resulted in a purer 500 mM fraction, the band was fainter and not as much protein was eluted. Subsequently, the third purification used a Binding buffer with a higher concentration of 20 mM imidazole. As it had already been determined that HSP22E and HSP22F bind very tightly to the column, the His-tagged proteins would still be able to bind while contaminants will be more likely to be washed off. Elution gradients were also increased to 50 mM - 500 mM imidazole. These higher concentrations allowed for more of the tightly binding HSP22E and HSP22F to be knocked off, resulting in a purer purification.</p>

				<p>Additionally, the HSP22E purification attempt #2 showed very high levels of background protein in all elution fractions. Though this was initially thought to be due to the concentration of imidazole in the elution buffer, the next attempt showed a similar result for HSP22F. It was determined that this was likely occurring due to contamination of the filter within the gravity column itself.</p>

				<h1 id="Assay">Characterisation and Assay</h1>
				<h2>Protein Concentration Assay</h2>
				<p>In order to exchange the elution buffer to a more suitable buffer for subsequent assays, dialysis in Snakeskin dialysis tubing (Thermofisher) was performed. However, the sHSP samples became insoluble and precipitated in all attempts. Therefore, due to time constraints and limited lab access, protein samples were kept in 325 mM imidazole elution buffer for all further experimentation. Subsequently, the Bradford assay  was performed to determine the concentration of the proteins while still in the imidazole elution buffer. All subsequent assays were designed to account for and normalise background imidazole within the sample. The Bradford assay was used to determine the concentrations of our proteins for our characterisation gels and chaperone activity assays.</p>

				<p>Bradford assay used Bradford reagent to visualise protein concentration, with measurements conducted on the SPECTROstar Nano absorbance reader (BMG Labtech) at 450nm and 590nm. The first attempt for the Bradford assay was ineffective as high amounts of imidazole in the buffer reacted with the Bradford reagent and resulted in overestimation of protein concentration. The assay was re-designed to account for background levels of imidazole. It was performed using PBS to dilute the samples and as a blank. A standard curve was constructed using bovine serum albumin (BSA) of known concentration. The amount of imidazole was normalised by measuring the absorbance of the same volume of 325 mM imidazole elution buffer and subtracting this from the HSP22E and HSP22F samples. Using the standard curve, the concentrations were determined to be 1.23 μM for the HSP22E and 2 μM for the HSP22F (Figure 6).</p> 

				<div class="figures">
					<img src="/assets/images/Engineering/WIKI FINAL.png">
					<p><b>Figure 6:</b> Graphs from Bradford assay. The y-axis represents the absorbance as a ratio between 590nm to 450nm to measure the ratio of blue to yellow in each sample. The equation of the trendline and R2 values are displayed in each graph. A) BSA Standard curve, which is produced using known amounts of BSA  ranging from 0 - 4μg, diluted in PBS), B) HSP22E Protein concentration assay, and C) HSP22F Protein concentration assay. The x-axis of both B) and C) refers to the volume of protein sample used in the assay. </p>
				</div>

				<h2>Characterisation of Complex Formation for HSP22E and HSP22F</h2>
				<p>The initial SDS-PAGE gels of purified proteins indicated two dark bands at around 37 kDa and 70 kDa. It was considered likely that these were our purified proteins self-dimerising through disulfide bonds between cysteine residues within the protein. To confirm this, pooled fractions of HSP22E and HSP22F were visualised on an SDS-PAGE gel (figure 7) both with and without dithiothreitol (DTT). DTT was used to reduce cysteines and prevent dimer formation through disulfide interactions. HSP22E and HSP22F samples were also incubated together in one fraction and visualised on the same gels to further characterise their interactions.</p>

				<div class="figures">
					<img src="/assets/images/Engineering/6. sds and native .png">
					<p> 
					<b>Figure 7:</b> SDS and Native PAGE A) SDS-PAGE with Lane 1 containing the Thermo Scientific Spectra Multicolor High Range Protein Ladder. The lowest boxes indicate the monomeric HSP at around 37 kDa, whilst higher boxes indicate high order complexes around 70 and 140 kDa. The last 3 samples were incubated in 60mM dithiothreitol (DTT) to reduce disulfide bond formation. B) Native PAGE, Lane 1 contains Thermo Fisher NativeMark Unstained Protein Ladder. Higher order complexes evident at top of the gel, with a defined band at 720kDa for all samples.</p>
				</div>

				<p>Samples with DTT showed comparatively darker bands at 37 kDa, indicating monomeric conformation. Samples without DTT showed increasingly darker bands at around 70, 140, 200 kDa and higher, suggesting formation of higher order complexes. Additional visualisation of samples in their native state on a Native PAGE gel (figure 8) indicated similar formation of many higher order complexes, as seen in the smeared lanes for all three samples. The assembly of higher order complexes was consistent with observations in our structural modelling of the proteins. This modelling work built upon the protein structure model of the sHSPs in order to visualise external cysteine residues as well as hydrophobic regions. The presence of both confirms that disulfide and hydrophobic interactions are involved in sHSP formation of large complexes in monomeric as well as dimersed forms. </p>

				<div class="figures">
					<img src="/assets/images/Engineering/7. WIKI FINAL.gif">
					<p> 
					<b>Figure 8:</b> Structural models of heat shock proteins constructed in PyMOL. A) Model of HSP22F with cysteine residues labelled in blue. B) Model of a dimer formation between HSP 22E and 22F with hydrophobic residues highlighted in blue. </p>
				</div>

				<h2>Chaperone Activity Assay </h2>
				<p>Chaperone activity assay was performed to assess the holdase activity of HSP22E and HSP22F. This involved measuring the aggregation of heat-labile protein citrate synthase (CS) at 43°C in a 1:1 ratio with a protein chaperone and comparing it to CS aggregation in the absence of chaperones as a control. All sample mixtures were calculated as such to normalise background imidazole levels. The chaperone activity assay was repeated multiple times, each with protocol optimisation as we did not have access to the Lambda UV–Vis spectrometer equipped with a thermostatting and stirring PTP‐1 peltier system (PerkinElmer, Waltham, MA, USA) that is commonly used by researchers using this type of chaperone assay. (11) Iterations of the protocol design included heating samples within a microcentrifuge tube on a heat block and within a cuvette in a water bath. Absorbance measurements of the control (CS only samples) were taken on either a microtiter plate using a SPECTROstar Nano absorbance reader (BMG Labtech) or in a cuvette using a cell density meter model 40 (Thermofisher scientific). These iterations of the protocol failed due to erratic values deviating from expected results seen previously in the literature when conducting this assay. (11)</p>

				<p>The finalised and successful protocol involved measuring and heating the cuvette in the cuvette reader in the Nano absorbance reader. This chaperone activity assay protocol was favoured as it showed the highest level of protein aggregation in the CS only dataset in comparison to the second iteration. From Figure 9, the aggregation with chaperones with either HSP22E or HSP22F showed lower levels of aggregation compared to the CS only data set. This indicates that the protein chaperones are capable of reducing heat-induced protein aggregation.</p> 

				<div class="figures">
					<img src="/assets/images/Engineering/Chaperone activity assay.png">
					<p> 
					<b>Figure 9:</b> HSP22E and HSP22F are shown to decrease heat-induced aggregation of citrate synthase (CS). Chaperone activity was measured by comparing the aggregation of CS incubated at 43 °C without protein chaperones (◇), with HSP22E (●), or with HSP22F (Δ). Aggregation of CS resulting from thermal denaturation was quantified by the changes in absorbance at 500 nm over time on the SPECTROstar Nano absorbance reader (BMG Labtech).</p>
				</div>

				<h2>Discussion</h2>
				<p>Initially after protein purification, dialysis with snakeskin dialysis tubing was attempted in order to exchange the elution buffer to a more suitable buffer for subsequent assays. Dialysis was attempted using a 20 mM sodium phosphate (pH 8) buffer with both 150 mM and 500mM NaCl. These are the normal sodiums level for general physiological process and the marine environment (where host organisms of these genes reside), respectively. However, both attempts of dialysis failed as the protein appeared to aggregate in the buffer and precipitate.\. Due to limitation in size of sample and COVID 19 lab access restrictions, no further dialysis attempts were undertaken. Instead, the samples containing imidazole were used for subsequent assays. Assays were redesigned to account for possible interference from background imidazole and salt levels.</p>

				<p>The first chaperone activity assay was to compare the aggregation of CS in HEPES-KOH (pH 7.2) buffer in comparison to imidazole elution buffer. The HEPES-KOH (pH 7.2) buffer was used as a point of comparison to a similar assay in a paper published by Dr. Dominic Glover, where good aggregation was observed (11). This point of comparison was important because the elution buffer contains a high concentration of salt and imidazole that could have protein stabilising effects (12, 13) . This could prevent the aggregation of protein and lead to an invalid result. Our protocol showed that 0.5 μM CS had greater aggregation in the elution buffer compared to the HEPES buffer. However, there was no consistent trend with the absorbance fluctuating in values. This was due to the fact that aggregated protein is very sticky and tended to stick onto the edges of the microcentrifuge tube and in the pipette tips when aspirating and transferring samples to the microtitre plate. Furthermore, as a microtitre plate was used, accurate pipetting became crucial, which was difficult to do in a consistent timely manner. As the assay progressed the volume inside the microfuge tube would decrease, increasing the amount of aggregated protein along the interior of the microcentrifuge tube, which could not be detected by the microtitre plate reader. This led to decreased absorbance as time progressed.</p> 

				<p>The second iteration aimed to reduce the need for transferring the sample to reduce loss of aggregated CS and inconsistent values. By performing the assay in a cuvette, the sample did not need to be transferred. However, it meant the sample had to be heated in a water bath instead of a heat block which was not as efficient as transferring heat to the sample.</p> 
				<p>A parafilm seal was used to allow the sample to be mixed by inversion rather than a pipette tip to reduce the amount of aggregated protein lost. 3 mL of the sample was used to completely fill the cuvette to prevent the loss of CS aggregating on the sides of the cuvette where it could not be detected by the cell density meter. Due to the large volume, a lower concentration of 0.1 μM CS had to be used due to the limited supply. However, the amount of aggregation was not detectable using the cell density meter model 40 (Fisher scientific) as it was not sensitive and could only read to 2 decimal points. Also, it only read the absorbance at 600nm, which deviates from the recommended 500nm in the original paper (11). The time intervals between removing the cuvette from the water bath and measuring the absorbance caused temperature fluctuations. Subsequently, little to no aggregation was observed, though this could also be due to the lower CS concentration in comparison to the first iteration.</p>

				<p>The third iteration combined the principles of the first 2 attempts; 0.5 μM CS and 1 mL of sample was used. The CS only assay used the same volume of 325 mM imidazole elution buffer as it would of HSP22E to ensure the highest amount of imidazole buffer added did not prevent CS aggregation. To counteract the loss of aggregation of protein of the sides of the cuvette through inversion, a pipette was used to gently aspirate before each measurement to decrease the amount of aggregated protein lost. The samples were also heated and measured in the SPECTROstar Nano absorbance reader (BMG Labtech) to reduce temperature fluctuations between readings. The absorbance reader was more sensitive than the cell density meter.</p>

				<p>This produced the characteristic sigmoidal-like curve in the CS only sample as shown in previous studies, depicting good aggregation. (11) This meant the chaperones activity of HSP22E and HSP22F could be assessed in relation to the CS only sample. HSP22F had slightly higher aggregation in comparison to HSP22E, indicating lower chaperone activity, this could be due to HSP22F having a lower purity as shown in the protein purification gel in Figure 5 where contamination is evident.</p> 

				<p>Future work would involve the additional step of incubating 0.25 μM HSP22E and 0.25 μM HSP22F overnight at 4°C to form the complexes that potentially enhance chaperone activity before being assessed for protein aggregation.</p>

				<h2 id="Conclusions">Conclusions/Future Directions </h2>
				<p>The experiments above provide new insight into the novel HSP22E and HSP22F. It was confirmed that HSP22E and HSP22F run higher on protein gels at around 37 kDa instead of their expected 22 kDa. It was also found that they homodimerise and form higher order complexes through disulfide bond interactions. The chaperone activity was also assessed by comparing the aggregation of citrate synthase with and without HSPs. It was observed for the first time that recombinant HSP22E and HSP22F are able to prevent the aggregation of heat-labile proteins.</p>

				<p>Further characterisation of these novel proteins could include assessing the chaperone activity of HSP22E/HSP22F complexes along with measuring the binding affinities of sHSPs using surface plasmon resonance (SPR) to supplement modeling work. In addition, thermotolerance assays comparing growth of heat-stressed E. coli with and without novel heat shock proteins can be conducted to gain further understanding of sHSP functionality within a living system.</p> 

				<p>Due to time constraints of limited lab access, experimental design this year focused on this preliminary characterisation of HSP22E and 22F. In Phase II of the project, this knowledge will be utilised to engineer the expression of a ROS-regulated sHSPs within E. coli. Characterisation and validation of this will inform the implementation of a similar system within Symbiodinium, as outlined in further detail in project design.</p> 

				<div class="references">
					<p id="References">REFERENCES</p>
					<ol id="references">
						<li>Webster JM, Darling AL, Uversky VN, Blair LJ. Small heat shock proteins, big impact on protein aggregation in neurodegenerative disease. Frontiers in pharmacology. 2019;10.</li>
						<li>Basha E, O’Neill H, Vierling E. Small heat shock proteins and α-crystallins: dynamic proteins with flexible functions. Trends in Biochemical Sciences. 2012 Mar 1;37(3):106–17.</li>
						<li>Garrido C, Paul C, Seigneuric R, Kampinga HH. The small heat shock proteins family: The long forgotten chaperones. The International Journal of Biochemistry & Cell Biology. 2012 Oct 1;44(10):1588–92.</li>
						<li>Mogk A, Ruger-Herreros C, Bukau B. Cellular functions and mechanisms of action of small heat shock proteins. Annual review of microbiology. 2019 Sep 8;73:89-110.</li>
						<li>Lee GJ, Vierling E. A small heat shock protein cooperates with heat shock protein 70 systems to reactivate a heat-denatured protein. Plant Physiology. 2000 Jan 1;122(1):189-98.</li>
						<li>Kobayashi Y, Harada N, Nishimura Y, Saito T, Nakamura M, Fujiwara T, Kuroiwa T, Misumi O. Algae sense exact temperatures: small heat shock proteins are expressed at the survival threshold temperature in Cyanidioschyzon merolae and Chlamydomonas reinhardtii. Genome biology and evolution. 2014 Oct 1;6(10):2731-40.</li>
						<li>Great Barrier Reef Tours. 2020. Annual Great Barrier Reef Weather Overview. [online] Available at: <https://greatbarrierreeftourscairns.com.au/blog/annual-great-barrier-reef-weather-overview/#:~:text=Temperatures%20are%20pretty%20steady%20throughout,2010mm%20falling%20during%20the%20year.> [Accessed 22 October 2020]. </li>
						<li>Mierendorf RC, Morris BB, Hammer B, Novy RE. Expression and Purification of Recombinant Proteins Using the pET System. In: Rapley R, editor. The Nucleic Acid Protocols Handbook [Internet]. Totowa, NJ: Humana Press; 2000 [cited 2020 Oct 28]. p. 947–77. (Springer Protocols Handbooks).</li>
						<li>Seo JS, Lee Y-M, Park HG, Lee J-S. The intertidal copepod Tigriopus japonicus small heat shock protein 20 gene (Hsp20) enhances thermotolerance of transformed Escherichia coli. Biochemical and Biophysical Research Communications. 2006 Feb 17;340(3):901–8.</li>
						<li>Kato K, Goto S, Inaguma Y, Hasegawa K, Morishita R, Asano T. Purification and characterization of a 20-kDa protein that is highly homologous to alpha B crystallin. J Biol Chem. 1994 May 27;269(21):15302–9.</li>
						<li>Glover DJ, Clark DS. Oligomeric assembly is required for chaperone activity of the filamentous γ-prefoldin. The FEBS Journal. 2015;282(15):2985–97.</li>
						<li>Tsumoto K, Ejima D, Senczuk AM, Kita Y, Arakawa T. Effects of salts on protein-surface interactions: applications for column chromatography. J Pharm Sci. 2007;96(7):1677-90.</li>
					</ol>
				</div>
			</div>
		</div>
	</section>
