  <section class="protocols">
    <div class="protocol-title">
      <h1><a id="#Cloning">CLONING</a></h1>
    </div>
    <h2><a id="Gibson_Assembly"> Ligation by Gibson Assembly</a></h2>
    <ol>
      <li>Prepare linearised pET-19b plasmid backbone.</li>
        <ul>
          <li>Linearised pET-19b plasmid at a concentration of 10ng/μL was provided to us by Dr. Dominic Glover.</li>
        </ul>
      <li>Prepare insert gene.</li>
        <ul>
          <li>Order the designed gene inserts from gene synthesis company such as Integrated DNA Technologies (IDT).</li>
          <li>Resuspend dried pellet in 50 μL of MilliQ water.</li> 
          <li>Measure DNA concentration on the Nanodrop spectrophotometer.</li> 
        </ul>
      <li>Prepare the following 5x Isothermal Master Mix and store at -20°C.</li>
    </ol>

    <table id="protocols">
      <thead>
        <tr>
          <td colspan="2"><b>5x Isothermal Master Mix</b></td>
        </tr>
      </thead>

      <tr>
        <td>1 M Tris-HCl (pH 7.5)</td>
        <td>3 mL</td>
      </tr>

      <tr>
        <td>1 M MgCl<sup>2</sup></td>
        <td>300 μL</td>
      </tr>

      <tr>
        <td>100 mM dGTP</td>
        <td>60 μL</td>
      </tr>

      <tr>
        <td>100 mM dATP</td>
        <td>60 µl</td>
      </tr>

      <tr>
        <td>100 mM dTTP</td>
        <td>60 μL</td>
      </tr>

      <tr>
        <td>100 mM dCTP</td>
        <td>60 μL</td>
      </tr>

      <tr>
        <td>1 M DTT</td>
        <td>300 μL</td>
      </tr>

      <tr>
        <td>PEG-8000</td>
        <td>1.5g</td>
      </tr>

      <tr>
        <td>100 mM NAD</td>
        <td>300 μL</td>
      </tr>

      <tr>
        <td>Deionised water</td>
        <td>360 µL</td>
      </tr>

      <tr>
        <td>Total</td>
        <td>6 mL</td>
      </tr>
    </table>

    <ol start="4">
      <li>Prepare the Gibson Assembly Master Mix using the 5x Isothermal Master Mix, and store in 15 µL aliquots at -20°C.</li>
    </ol>

    <table id="protocols">
      <thead>
        <tr>
          <td colspan="2"><b>Gibson Assembly Master Mix </b></td>
        </tr>
      </thead>

      <tr>
        <td>5X Isothermal Master Mix</td>
        <td>320 μL</td>
      </tr>

      <tr>
        <td>10 U/μL T5 exonuclease</td>
        <td>0.64 μL</td>
      </tr>

      <tr>
        <td>2 U/μL Phusion DNA Pol</td>
        <td>20 μL</td>
      </tr>

      <tr>
        <td>40 U/μL T4 DNA Ligase</td>
        <td>0.16 μL </td>
      </tr>

      <tr>
        <td>Deionised water</td>
        <td>860 μL</td>
      </tr>

      <tr>
        <td>Total</td>
        <td>1.2 mL</td>
      </tr>
    </table>

    <ol start="5">
      <li>Add equimolar amounts of DNA insert and plasmid backbone to make up 5 μL. Small DNA inserts (<1000kb) should be added 5x in excess.</li> 
      <li>Add the 5 μL of DNA to 15 μL of the thawed Gibson Assembly Master Mix prepared above. Complete on ice.</li>
      <li>Incubate at 50°C for one hour.</li> 
    </ol>

    <h2><a id="Heat_Shock_Transformation">Heat Shock Transformation</a></h2>
      <h3><a id="Agar_Plates">Preparation of Ampicillin Agar Plates</a></h3>
      <ol>
        <li>Prepare 100 ml of LB agar and autoclave.</li>  
        <li>Allow to cool until it can be touched. Add 100 μL of 1000x Ampicillin stock.</li> 
        <li>Pour into plates under sterile conditions and wait until agar sets</li> 
      </ol>

      <h3><a id="Transformation">Transformation</a></h3>
      <ol>
        <li>Add 20 μL of Assembled Gibson DNA to 50 μL of competent E. coli BL21 (DE3) on ice.</li> 
        <li>In parallel, prepare two control tubes on ice, each containing the 50 μL of competent E. coli BL21 (DE3) added to:</li>  
        <ul>
          <li>Negative Control 1: 5 μL of deionised water.</li> 
          <li>Negative Control 2: Plasmid backbone at the equivalent concentration to that added to the Gibson assembly.</li> 
        </ul>
        <li>Heat shock at 42°C for 45 seconds.</li>  
        <li>Immediately place on ice for 2 minutes.</li>  
        <li>Add 1 mL of LB under sterile conditions and leave in the shaking incubator at 200rpm 37°C for 1 hour.</li> 
        <li>Centrifuge cells 15,000rpm for 2 minutes to obtain pellets. Discard around 950 μL of supernatant without dislodging pellet, and resuspend in the remaining amount.</li>  
        <li>Spread plate 50 μL on Ampicillin agar plates, incubate overnight at 37°C.</li> 
      </ol>

      <h2><a id="Patch_Glycerol">Patch Plates and Glycerol Stocks</a></h2>
        <h3><a id="Patch">Patch Plates</a></h3>
        <ol>
          <li>Prepare ampicillin selection plates as above.</li> 
          <li>Divide plates into small boxes.</li> 
          <li>Touch sterile pipette tip to a colony and streak in a small patch within a box.</li> 
          <li>Label each box carefully.</li> 
          <li>Incubate overnight at 37°C and store at 4°C for later use.</li>  
        </ol>

        <h3><a id="Glycerol">Glycerol Stocks</a></h3>
        <ol>
          <li>Prepare a mix of 50% glycerol and 50% LB upto 10 mL in a falcon tube under sterile conditions. Invert tube until mixed.</li> 
          <li>Filter with 0.22μm filter under sterile conditions.</li>  
          <li>Pipette 500 μl of LB/glycerol and 500 μl overnight culture into a cryotube under sterile conditions.</li> 
          <li>Freeze at -80°C.</li> 
        </ol>

      <h2><a id="Colony_PCR">Colony PCR</a></h2>
      <ol>
        <li>Design and order PCR forward and reverse primers that bind to either end of the plasmid backbone in order to amplify the insert.</li> 
        <ul>
          <li>The following primers were used to amplify insert within a pET-19b backbone:</li>   
          <li>Fwd Primer: 5'-ATGCGTCCGGCGTAGA</li> 
          <li>Rev Primer: 5'-CCGTTTAGAGGCCCCAAG</li>
        </ul>
        <li>Prepare PCR Master Mix according to the number of colonies being screened.</li> 
      </ol>

    <table id="protocols">
      <thead>
        <tr>
          <td><b>Reagent</b></td>
          <td><b>Single Amount</b></td>
          <td><b>Master Mix (10 volumes)</b></td>
        </tr>
      </thead>

      <tr>
        <td>Water</td>
        <td>4.6μL</td>
        <td>46μL</td>
      </tr>

      <tr>
        <td>Taq 2x Master Mix with loading dye</td>
        <td>5μL</td>
        <td>50μL</td>
      </tr>

      <tr>
        <td>T7 forward primer</td>
        <td>0.2μL</td>
        <td>2μL</td>
      </tr>

      <tr>
        <td>T7 reverse primer </td>
        <td>0.2μL</td>
        <td>2 μL</td>
      </tr>

      <tr>
        <td>Total</td>
        <td>10 μL</td>
        <td>100 µl</td>
      </tr>
    </table>

    <ol start="3">
      <li>Aliquot 10μL of PCR Master Mix into PCR tubes according to the number of colonies being screened.</li>  
      <li>Inoculate the PCR Master Mix with the colony, using a sterile pipette tip or inoculating loop to gently touch and transfer. Patch plates can be taken at the same time, as described above.</li> 
      <li>Thermocycle under the following conditions. Set the lid temperature to 105°C. Repeat Denaturation, Annealing and Extension Cycle 34x.</li> 
    </ol>

    <table id="protocols">
      <thead>
        <tr>
          <td colspan="3"><b>PCR Thermocycler conditions. </b></td>
        </tr>
      </thead>

      <tr>
        <td>PCR Cycle Step </td>
        <td>Temperature</td>
        <td>Time</td>
      </tr>

      <tr>
        <td>Initial denaturation</td>
        <td>95°C</td>
        <td>5 min </td>
      </tr>

      <tr>
        <td>Denaturation </td>
        <td>95°C</td>
        <td>30 sec </td>
      </tr>

      <tr>
        <td>Annealing*</td>
        <td>55°C</td>
        <td>30 sec</td>
      </tr>

      <tr>
        <td>Extension** </td>
        <td>68°C</td>
        <td>2 min</td>
      </tr>

      <tr>
        <td>Final Extension</td>
        <td>68°C</td>
        <td>5 min</td>
      </tr>

      <tr>
        <td>Hold</td>
        <td>4°C</td>
        <td>Hold</td>
      </tr>
    </table>

    <ul id="references">
      <li>*Note: Annealing temperature is dependent upon primer design.</li>  
      <li>**Note: Extension time is dependent on length of gene of interest.</li> 
    </ul>

    <h2><a id="Agarose_Gel">Agarose Gel Electrophoresis</a></h2>
    <ol>
      <li>Prepare 50 mL of 0.8% agarose with TBE buffer. TBE buffer is used for high resolution of shorter DNA fragments (<1000bp).</li>  
      <li>Weigh the flask and microwave for 45 seconds, or until fully dissolved.</li>  
      <li>Allow agarose to cool and then reweigh. Add water to make up to initial weight.</li>  
      <li>Pour agarose into mould, ensuring that comb has been inserted to form the wells. Allow gel to set, and then carefully remove comb.</li>   
      <li>Place gel into gel tank, ensuring that the wells are aligned on the side of the negative electrode. Pour TBE buffer until gel is fully covered.</li>   
      <li>Load 5 μL of 1kb Plus DNA Ladder into the first well.</li>  
      <li>Prepare 10 μL of each sample added to 2 μL of Loading dye. Load into subsequent wells.</li>  
      <li>Run at 100V for 50 minutes, ensuring small bubbling is visible at electrodes. Dye should slowly migrate over time towards the positive electrode.</li>   
      <li>Remove from the tank and place into a square petri dish.</li>   
      <li>Wash once with water, leave on rocking platform for 10-15 minutes.</li>   
      <li>Post stain with 4 μL of SYBR Gold stain added to enough TBE to cover wells. Place on a rocking platform for 25 minutes to stain.</li>   
      <li>Image using gel under the transilluminator setting with GelDocs using the recommended protocol for DNA gels and SYBR Gold stain.</li>   
    </ol>

    <h2><a id="Miniprep">Plasmid Extraction and Purification</a></h2>
    <p>The centrifugation processing protocol in <b><a id="protocol-link" href="https://www.qiagen.com/au/resources/resourcedetail?id=e0fab087-ea52-4c16-b79f-c224bf760c39&lang=en">Qiagen QIAquick PCR Purification Kit</a></b> was followed, without any changes. Sample was eluted in 50 μL water.</p>

    <h2><a id="Sequencing">Sequencing</a></h2>
    <ol>
      <li>Add 8 μL of purified DNA and 2 μL of either forward or reverse primer into a microcentrifuge tube.</li> 
      <li>Request sequencing at the Ramaciotti Centre for Genomics via the online portal, and label each tube following instructions with the order number.</li> 
      <li>Submit samples to the Ramaciotti Centre for Genomics for sequencing.</li> 
    </ol>

  <div class="protocol-title">
    <h1><a id=“Purification”>PROTEIN EXPRESSION AND PURIFICATION</a></h1>
  </div>
    <h2><a id="Induction">Induction</a></h2>
      <h3><a id="Starter">Starter Culture</a></h3>
      <ol>
        <li>Set up starter cultures with 5 mL of Luria Broth (LB) and 5 μL of 1000X ampicillin.</li> 
        <li>Innoculate with colonies from patch plates using a pipette tip.</li> 
        <li>Incubate overnight in the shaking incubator at 37°C and 200rpm.</li>  
      </ol>

      <h3><a id="Induction_Culture">Induction Culture</a></h3>
      <ol>
        <li>Prepare larger cultures with 500 mL LB and 500 μL of 1000x ampicillin.</li>  
        <li>Innoculate with 1 mL of starter culture. Incubate until OD600 reaches 0.6.</li>  
        <li>Separate 500 mL cultures into 2x 250 mL cultures.</li>
        <li>Remove the equivalent of 1 mL of culture at 0.6 OD600 for Expression check.</li> 
        <li>Centrifuge at 13000rpm for 1 minute, remove supernatant and freeze pellet at -20°C.</li> 
        <li>Add 0.5mM of IPTG to the larger cultures to induce expression.</li> 
        <li>Incubate in the shaking incubator at 200rpm overnight at 20°C.</li> 
      </ol>

    <h2><a id="Harvesting">Harvesting Cells</a></h2>
    <ol>
      <li>Pour 250 mL induction cultures into 1L centrifuge containers.</li>  
      <li>Centrifuge for 20 minutes at 5000rpm at 4°C to form pellets.</li>  
      <li>Pour out most of the supernatant, leaving around 30 mL. Resuspend bacterial pellet.</li> 
      <li>Transfer resuspended pellet into 50mL Falcon tubes. Add water to balance tubes if necessary.</li>  
      <li>Centrifuge tubes at 4000rpm for 20 minutes at 4°C. Remove all supernatant and freeze cells for future work.</li>  
    </ol>

    <h2><a id="BugBuster">Expression Check (BugBuster)</a></h2>
    <ol>
      <li>Collect frozen pellet with ‘Before induction’ sample (Step 3 of <b><a id="protocol-link" href="/Protocols/#Induction_Culture"> Induction protocol</a></b>).</li> 
      <li>Collect a ‘After induction’ sample with the equivalent of 1 mL at an OD600 of 1. Centrifuge at 13000rpm for 1 minute and remove supernatant.</li> 
      <li>Add 100 μL BugBuster Protein Extraction Reagent to Before and After samples. Vortex and wait 5 min until dissolved. Centrifuge for 20 minutes at 14,000rpm.</li> 
      <li>Transfer supernatant from the ‘Before induction’ sample to another microcentrifuge tube. This will be the ‘Before induction’ fraction to be visualised later.</li> 
      <li>Transfer supernatant from the ‘After induction’ sample to another microcentrifuge tube. This will be the ‘Soluble after induction’ fraction to be visualised later.</li>  
      <li>To prepare the ‘Insoluble after induction’ sample, make up a BB mixture of 1:10 bugbuster:water. Wash pellet from the ‘After induction’ sample with 600 μL of the BB mix. Vortex and centrifuge for 15 minutes at 14 000rpm. Discard supernatant and resuspend in 100 μL of BB mix. This will be the ‘Insoluble after induction’ sample.</li>  
      <li>Run all samples on SDS-PAGE gel.</li> 
    </ol>

    <h2><a id="SDS_PAGE">SDS-PAGE</a></h2>
      <h3><a id="Preparing_Gel">Preparing Gel</a></h3>
        <ol>
          <li>Assemble the SureCast Gel Handcast System with a silicone spacer in between two glass plates clamped into place.</li> 
          <li>Prepare Separation gel mixture as follows. Scale up if making more than one gel.</li> 
        </ol>

      <table id="protocols">
      <thead>
        <tr>
          <td colspan="2"><b>Seperation gel</b></td>
        </tr>
      </thead>

      <tr>
        <td>Water</td>
        <td>2.87 mL</td>
      </tr>

      <tr>
        <td>Acrylamide:bis (37.5:1) 30%</td>
        <td>2.31 mL</td>
      </tr>

      <tr>
        <td>Tris-HCl 1.5M pH 8.8 </td>
        <td>1.75 mL</td>
      </tr>

      <tr>
        <td>SDS 10%</td>
        <td>70 μL</td>
      </tr>

      <tr>
        <td>TEMED</td>
        <td>8.4 μL</td>
      </tr>

      <tr>
        <td>APS 10%</td>
        <td>63 μL</td>
      </tr>

      <tr>
        <td>Total</td>
        <td>~7 mL</td>
      </tr>
    </table>

    <ol start="3">
      <li>Pipette mixture immediately into the glass plates assembly until the first fill line. Add 500 μl of isopropanol on top to ensure the surface of the gel is even. Polymerisation should occur within 15-30 minutes. Leave to sit until set.</li>  
      <li>Pour out isopropanol with the system still assembled, and rinse two times with water. Remove any remaining water with absorbent tissue paper.</li>  
      <li>Prepare Stacking gel mixture as follows. Scale up if making more than one gel</li> 
    </ol>

    <table id="protocols">
    <thead>
      <tr>
        <td colspan="2"><b>Stacking gel</b></td>
      </tr>
    </thead>

    <tr>
      <td>Water</td>
      <td>1.2 mL</td>
    </tr>

    <tr>
      <td>Acrylamide:bus (37.5:1) 30%</td>
      <td>260 μL</td>
    </tr>

    <tr>
      <td>Tris-HCl 1.5M pH 8.8</td>
      <td>500 μL</td>
    </tr>

    <tr>
      <td>SDS 10%</td>
      <td>20 μL</td>
    </tr>

    <tr>
      <td>TEMED</td>
      <td>1.8 μL</td>
    </tr>

    <tr>
      <td>APS 10%</td>
      <td>12 μL</td>
    </tr>

    <tr>
      <td>Total</td>
      <td>~2 mL</td>
    </tr>
  </table>

  <ol start="6">
    <li>Pipette mixture immediately on top of the separating gel, fill to the top of the plates.</li> 
    <li>Gently place comb in between glass plates to form wells. Push bubbles to the side to remove them. Let sit until set.</li>
  </ol>

  <h3><a id="Samples">Preparing and Loading Samples</a></h3>
  <ol>
    <li>Unclamp assembly system and remove gel along with glass plates. Remove the comb gently.</li>
    <li>Move to the assembled gel tank and clamp between electrodes with wells facing the front of the tank.</li> 
    <li>Pour MES buffer into tank. All wells should be covered in front. Open glass slit at the back should also be covered.</li> 
    <li>Load 5 μL of protein standard ladder into the first well.</li> 
    <li>Prepare sample by adding 5 μL of SDS Loading Dye (6x) and 20 μL of sample.</li>
    <li>Incubate samples for 10 minutes at 95°C. Load 20 μL of sample into subsequent wells.</li>
  </ol>

  <h3><a id="Electrophoresis">Electrophoresis</a></h3>
  <ol>
    <li>Place the lid onto the tray and plug into the power pack.</li>
    <li>Run gel at 180V for 30 minutes.</li> 
    <li>After running, turn off the power pack and remove gel with glass plates from the tray.</li>
    <li>Gently separate glass plates and remove gel. Place in a square petri dish.</li>
    <li>Wash with water and leave to sit in water on the rocking platform for 10 minutes. Repeat twice.</li>
  </ol>

  <h3><a id="Staining">Gel Staining</a></h3>
  <ol>
    <li>Add Coomassie blue stain so that it just covers the gel. Leave to stain on the rocking platform for at least 20 minutes or until bands are clearly visible.</li>
    <li>Remove Coomassie blue stain and destain with water overnight.</li>
    <li>Image gel under the transilluminator setting with GelDocs using the recommended protocol for protein gels and Coomassie blue stain.</li>
  </ol>

  <h2><a id="Sonication">Cell Lysis by Sonication</a></h2>
  <ol>
    <li>Resuspend harvested pellet in 30 mL of Binding Buffer used for purifications.</li>
    <li>Sonication on ice for 8 minutes with 2 second on/off intervals at 60% amplitude.</li>
    <li>Centrifuge for 20 minutes at 4000rpm and 4°C to obtain a soluble protein sample within supernatant. This will be used for purification.</li>
  </ol>

  <h2><a id="Purification">Ni-NTA Purification</a></h2>
    <h3><a id="Buffers">Purification: Preparing Buffers and Samples</a></h3>
    <ol>
      <li>Prepare Binding Buffer stock solution with 10mM imidazole, 500mM NaCl, 50mM sodium phosphate buffer to a pH of 8.</li> 
      <li>Prepare Elution Buffer stock solution with 500mM imidazole, 500mM NaCl, 50mM sodium phosphate buffer to a pH of 8.</li> 
      <li>Filter buffers.</li>
      <li>Prepare 2 mL of Elution Buffer gradients stock solutions at the following concentrations: 50mM, 60mM, 70mM, 80mM, 90mM, 100mM, 120mM, 140mM, 200mM, 250mM, 300mM, 350mM, 450mM, 500mM. This can be done by calculating and mixing Binding Buffer with Elution Buffer.</li> 
      <li>Prepare sample by filtering sonication supernatant with 0.22μm filter.</li> 
    </ol>

    <h3><a id="Calibration">Purification: Calibrating Resin</a></h3>
    <ol>
      <li>Add 2 mL of resuspended Ni-NTA resin slurry to a 50 mL falcon tube.</li>  
      <li>Calibrate resin by adding 50 mL of Binding Buffer and mixing.</li>  
      <li>Centrifuge at 1000rpm for 5 minutes at 4°C. Ensure acceleration and deceleration speeds are set at 5.</li>
      <li>Remove supernatant without disturbing resin.</li>
      <li>Repeat three times.</li> 
    </ol>

    <h3><a id="Incubation">Purification: Protein Incubation</a></h3>
    <ol>
      <li>Remove 1 mL of filtered Lysate sample and store in microcentrifuge to visualise purification on SDS-PAGE later.</li>  
      <li>Incubate resin with 30 mL filtered sample on a rotating block at 4°C for one hour.</li> 
      <li>Centrifuge at 1000rpm for 5 minutes at 4°C. Ensure acceleration and deceleration speeds are set at 5.</li> 
      <li>Collect supernatant in a falcon tube as the Flowthrough sample. Keep on ice.</li>  
    </ol>

    <h3><a id="Wash">Purification: Wash</a></h3>
    <ol>
      <li>Add 50 mL of Binding Buffer and incubate resin on a rotating block for 30 minutes.</li> 
      <li>Collect supernatant in a falcon tube as the Wash sample. Keep on ice.</li> 
      <li>Repeat three times, collecting and pooling supernatant each time.</li> 
      <li>Resuspend resin in 5 mL of Binding Buffer and transfer to gravity column. Allow Binding Buffer to flow through while ensuring that the resin does not dry out.</li>  
    </ol>

    <h3><a id="Elution">Purification: Elution</a></h3>
    <ol>
      <li>Sequentially add 2 mL Elution Buffer gradients, collecting each separate fraction in a 2 mL tube.</li> 
      <li>After collecting all elution fractions, wash the resin within the column with 5 ml water, 5 mL 0.5M NaOH and 5 mL water.</li> 
      <li>Store resin in 20% ethanol at 4°C.</li> 
    </ol>

    <div class="protocol-title">
      <h1><a id="Assay">CHARACTERISATION AND ASSAY</a></h1>
    </div>
    <h2><a id="Bradford">Protein Concentration Assay (Bradford)</a></h2>
    <ol>
      <li>Prepare a stock solution of bovine serum albumin (BSA) stock solution at 0.1mg/mL.</li> 
      <li>Dilute Bradford reagent (Bio-Rad) 2.5-fold in water.</li>  
      <li>Add 200 μL of water in triplicate wells on a microtiter plate. This will be the blank.</li> 
      <li>Add 0, 10, 20, 30, 40 and 50 μL of BSA stock solution in triplicate. Add water to make up 100 μL in each well.</li>  
      <li>Add 0, 10, 20, 30, 40 and 50 μL of unknown sample in triplicate. Add water to make up 100 μL in each well.</li>  
      <li>Add  0, 10, 20, 30, 40 and 50 μL of Elution Buffer (325mM imidazole, 500mM NaCl, 50mM sodium phosphate buffer to a pH of 8). Add water to make up 100 μL in each well. This will be used to normalise the amount of imidazole in unknown sample wells as the presence of imidazole affects colour development </li>  
      <li>Add 100 μL of diluted Bradford reagent to every well except for the blank. Mix well. Wait at least 5 minutes but no longer than 60 minutes for colour development.</li> 
      <li>Measure absorbance at 590nm and 450nm on SPECTROstar Nano absorbance reader (BMG Labtech).</li>  
      <li>Prepare standard curve by subtracting average blank values from average BSA absorption values at both 590 nm and 450 nm. Divide 590 nm by 450 nm and plot graph to obtain the linear trendline.</li>  
      <li>Plot unknown sample values by subtracting the average blank as well as average background imidazole absorbances from the sample absorbance. Divide 590 nm by 450 nm values and obtain the linear trendline.</li>  
      <li>To find the concentration of the sample in mg/mL, divide the slope of the unknown graph by the slope of the BSA standard curve.</li> 
    </ol>

    <h2><a id="Native">Native PAGE.</a></h2>
    <ol>
      <li>Remove Tris-Glycine 4-12% gel (Novex WedgeWell 1.0mm Protein Gel from Invitrogen) from packaging and gently wash with water. Carefully remove the comb and the white tape at bottom.</li> 
      <li>Clamp gel with plastic plates between electrodes. Pour TBE buffer into tank. All wells should be covered in front, while the open glass slit at the back should be covered.</li>
      <li>Load 5 μL of Native protein standard ladder into the first well. Load 20 μL of samples into subsequent wells.</li>
      <li>Run gel at 70V for 210 minutes at 4°C.</li>
      <li>After running, turn off the power pack and remove gel from the tray.</li> 
      <li>Crack open plastic plates and remove gel. Place in a square petri dish. Wash with water and leave to sit in water on the rocking platform for ten minutes.</li> 
      <li>Add Coomassie blue stain so that it just covers the gel. Leave to stain on the rocking platform for at least 20 minutes or until bands are clearly visible.</li> 
      <li>Remove Coomassie blue stain and destain with water overnight.</li>
      <li>Image gel under the transilluminator setting with GelDocs using the recommended protocol for protein gels and Coomassie blue stain.</li> 
    </ol>

    <h2><a id="CS_Assay">Citrate Synthase Assay</a></h2>
    <ol>
      <li>Prepare the following 3 samples below in separate plastic cuvettes using the amounts shown below:</li> 
    </ol>

    <table id="protocols">
    <thead>
      <tr>
        <td colspan="2"><b>CS Only sample</b></td>
      </tr>
    </thead>

    <tr>
      <td>325μM imidazole elution buffer</td>
      <td>406.5 μL </td>
    </tr>

    <tr>
      <td>Water</td>
      <td>588.7 μL</td>
    </tr>
    </table>

    <ol start="2">
      <li>Incubate sample in SPECTROstar Nano absorbance reader (BMG Labtech) at 43°C for 10 minutes.</li> 
      <li>Measure the absorbance of the blank.</li> 
      <li>Add 4.8 μL of 103.529μM citrate synthase ammonium sulfate suspension into the cuvette</li> 
      <li>Aspirate sample gentle using pipette and immediately measure the absorbance at 500nm for time point 0 minutes.</li>  
      <li>Repeat step 5, every 5 minutes for 110 minutes. Absorbance of blank is automatically subtracted from absorbance readings by SPECTROstar Nano absorbance reader (BMG Labtech).</li> 
      <li>Repeat steps 2-6 for the HSP22E and HSP22F samples using the amounts shown below:</li>  
    </ol>

    <table id="protocols">
    <thead>
      <tr>
        <td colspan="2"><b>HSP22E sample</b></td>
      </tr>
    </thead>

    <tr>
      <td>1.23μM HSP22E  </td>
      <td>406.5 μL </td>
    </tr>

    <tr>
      <td>Water</td>
      <td>588.7 μL</td>
    </tr>
    </table>

   <table id="protocols">
    <thead>
      <tr>
        <td colspan="2"><b>HSP22F sample</b></td>
      </tr>
    </thead>

    <tr>
      <td>2 μM HSP22F</td>
      <td>250 μL</td>
    </tr>

    <tr>
      <td>Water</td>
      <td>745.2 μL</td>
    </tr>
    </table>

    <ol start="8">
      <li>Graph absorbance values of all 3 samples on graphing software GraphPad Prism 8.</li>  
    </ol>
</section>